sympy/printing/pycode.py

```diff
@@ -488,11 +484,10 @@ class NumPyPrinter(PythonCodePrinter):
     _kc = {k: 'numpy.'+v for k, v in _known_constants_math.items()}
 
 
-    def _print_seq(self, seq):
+    def _print_seq(self, seq, delimiter= ', '):
         "General sequence printer: converts to tuple"
         # Print tuples here instead of lists because numba supports
         #     tuples in nopython mode.
-        delimite.get('delimiter', ', ')
```

Comment:
delimite not define, should probably have been something like `delimiter = kwargs.get(...`

Should be reasonable to add a test for.

Reply:
I added a test for this. However, it does not seem like any existing code uses it.